oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+web
```

Comment:
this file is not needed anymore in recent OCA branch.

Reply:
Removed